README.md

```diff
@@ -46,7 +50,7 @@ Example for local development on the host **http://localhost:3000** and staging
 
 1.  Get the auth0 application values: **DOMAIN, CLIENT_ID, CLIENT_SECRET**
 
-1.  Create file **.env.local** and fill it with information:
+1.  Create file **.env** using template **.env.example** and fill it with auth0 information:
```

Comment:
Let’s keep all sensitive vars in `.env.local`. If we do this, `.env` can be checked into git. This is the recommended way, according to Next.js docs:

https://nextjs.org/docs/basic-features/environment-variables#loading-environment-variables

Reply:
Done